pkgs/development/compilers/ldc/default.nix

```diff
@@ -83,7 +83,7 @@ stdenv.mkDerivation rec {
   ++ stdenv.lib.optional (!bootstrapVersion && stdenv.hostPlatform.isDarwin) [
     # https://github.com/NixOS/nixpkgs/issues/57120
     # https://github.com/NixOS/nixpkgs/pull/59197#issuecomment-481972515
-    llvm
+    llvm_5
```

Comment:
Maybe one more comment above to explain why, link to last relevant issue, and explaining it can probably be changed to default LLVM7 on update to 1.16 ?

Reply:
I don't think that is necessary because it's just the bootstrap compiler which needs to use llvm5, the final ldc already uses llvm8.
This bootstrap compiler needs to go away soon anyway as it is already the case that compiling ldc with that leads to a compiler without all the possible features now.
Because of that I don't think the ldc developers will put a lot of effort into backporting stuff to the bootstrap branch and that would be necessary to fix that bug.
I already planned to get rid of this bootstrap compiler.